### PR TITLE
Fixed shared field names between measures and points

### PR DESCRIPTION
**Pull request by jessemapel**
Shared field names between measures and points are now called `measure_` and `point_`

Until #121 is fixed, I have removed the changes to the `to_isis` method.

### CODE REVIEW

File: plio/io/io_controlnetwork.py [hunk 1]
```diff
@@ -180,10 +199,11 @@ def read(self):
...
 
                 byte_count += 4 + message_size
-        self.point_attrs = [i if i != 'jigsawRejected' else 'pointJigsawRejected' for i in self.point_attrs]
-        cols = self.point_attrs + self.measure_attrs
 
-        cols = self.point_attrs + self.measure_attrs
+        # Some point and measure fields have the same name, so mangle them as point_ and measure_
+        point_cols = [self.point_field_map[attr] if attr in self.point_field_map else attr for attr in self.point_attrs]
+        measure_cols = [self.measure_field_map[attr] if attr in self.measure_field_map else attr for attr in self.measure_attrs]
+        cols = point_cols + measure_cols
```
Comment (jlaura) [inline plio/io/io_controlnetwork.py]: :+1:

**Comment by jessemapel** [comment]
This is going to have knock on effects in autocnet. So once this is merged, hold off on updating!

**Comment by jessemapel** [comment]
I'm not sure why coveralls is complaining about the coverage going down. I didn't modify the constructor property at all.

**Comment by jlaura** [comment]
@jessemapel Are we good to merge this? If so, anyone can do it. Reviewed and code looks good.

**Comment by jessemapel** [comment]
@jlaura I want to wait until https://github.com/USGS-Astrogeology/autocnet/pull/349 is finished because this will break that.

**Pull request merged**